server/nodeDef/nodeDefApi.js

```diff
@@ -40,12 +44,15 @@ module.exports.init = app => {
   app.put('/nodeDef/:id/prop', async (req, res) => {
     const {user, body} = req
 
+    const {key, value} = body
+
     const nodeDefId = getRestParam(req, 'id')
 
     try {
       await updateNodeDefProp(nodeDefId, body)
-
-      sendOk(res)
+      const nodeDef = await fetchNodeDef(nodeDefId, true)
```

Comment:
you don' t need to fetch the nodeDef here. the updateNodeDefProp already returns the updated nodeDef.
this would be instead
```javascript
const nodeDef = await updateNodeDefProp(nodeDefId, body)
```

Reply:
Ok